homeassistant/components/rest/data.py

```diff
@@ -62,6 +62,8 @@ async def async_update(self, log_errors=True):
                 data=self._request_data,
                 timeout=self._timeout,
             )
+            if not response.encoding:
```

Comment:
This will break any situation where the previous default encoding was the right one?

Reply:
No, not really. `utf-8-sig` is a fallback for normal `utf-8` so it should be safe like this..

This is how httpx decodes: https://github.com/encode/httpx/blob/master/httpx/_decoders.py#L249